app/helpers/alchemy/admin/pages_helper.rb

```diff
@@ -1,3 +1,4 @@
+# coding: utf-8
```

Comment:
That was added automatically by Emacs. Feel free to remove it, although in my opinion it's not a bad idea to keep it there. Some editors don't detect file encoding automatically.